tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -1123,6 +1123,40 @@ def testResourceRejection(self):
                                                "cannot be a resource variable"):
         guarantee_a.eval()
 
+class BroadcastToTest(test_util.TensorFlowTestCase):
```

Comment:
I think it'd be nice to have these tests in their own file, as this file is already quite long.

Reply:
Done. Thanks.